calculatorBackend/test/calculator_test[1]_include.cmake

```diff
@@ -0,0 +1,5 @@
+if(EXISTS "/home/jared/CalculatorSkeleton/calculatorBackend/test/calculator_test[1]_tests.cmake")
```

Comment:
I don't believe these are needed in github. These are created when the program is built